marklogic-data-hub-central/ui/src/components/flows/flows.tsx

```diff
@@ -150,6 +150,7 @@ const Flows: React.FC<Props> = (props) => {
                                 event.stopPropagation(); // Do not trigger collapse
                                 handleFlowDelete(name);
                             }}
+                            data-testid={'deleteFlow-' + i}
```

Comment:
same comment applies here. Can we try using getByLabelText and maybe replace row number "i" in line 145 above and line 159 below with flow/stepName if possible.